database/ticket.go

```diff
@@ -189,8 +288,7 @@ func (vdb *VspDatabase) GetTicketByHash(ticketHash string) (Ticket, bool, error)
 }
 
 // CountTickets returns the total number of voted, revoked, and currently voting
-// tickets. Requires deserializing every ticket in the db so should be used
-// sparingly.
+// tickets. Iterates over every ticket in the db so should be used sparingly.
```

Comment:
s / Iterates ... / This Iterates ... or It Iterates ...

Reply:
Updated